gitwash/development_workflow.rst

```diff
@@ -333,6 +383,97 @@ If you forgot to make a backup branch::
    # reset the branch to where it was before the botched rebase
    git reset --hard cool-feature@{2}
 
+
+.. _force_push:
+
+Force Push
+----------
+
+Your branch is now rebased, however, because of the way git
```

Comment:
This isn't really true, it shares all the commits up until the branch point.  Can you think of a way of rephrasing, that gives the idea that this is not fast-forward, and therefore needs a force push?